docs/driver-parameters.md

```diff
@@ -36,6 +36,9 @@ mountPermissions | mounted folder permissions. The default is `0777`, if set as
 vnetResourceGroup | specify vnet resource group where virtual network is | existing resource group name | No | if empty, driver will use the `vnetResourceGroup` value in azure cloud config file
 vnetName | virtual network name | existing virtual network name | No | if empty, driver will use the `vnetName` value in azure cloud config file
 subnetName | subnet name | existing subnet name of the agent node | No | if empty, driver will use the `subnetName` value in azure cloud config file
+softDeleteBlobs | Enable soft delete for blobs, specify the days to retain deleted blobs | "7" | No | Soft Delete Blobs is disabled if empty
```

Comment:
it's better providing a link to the storage doc to explain these parameters and what's the allowed values.